i18n/ko.json

```
@@ -1,1962 +1,1962 @@
+    "transcoding_audio_codec": "오디오 코덱",
+    "transcoding_audio_codec_description": "Opus는 가장 뛰어난 품질을 제공하지만, 구형 장치나 소프트웨어와의 호환성이 낮을 수 있습니다.",
+    "transcoding_bitrate_description": "최대 비트레이트를 초과하거나 허용 포맷이 아닌 영상",
+    "transcoding_codecs_learn_more": "용어에 대한 자세한 설명은 FFmpeg 문서를 참조하세요: <h264-link>H.264</h264-link>, <hevc-link>HEVC</hevc-link>, <vp9-link>VP9</vp9-link>",
+    "transcoding_constant_quality_mode": "고정 품질 모드",
+    "transcoding_constant_quality_mode_description": "ICQ가 CQP보다 우수하지만 일부 하드웨어는 지원하지 않습니다. 이 옵션을 설정하면 품질 기반 인코딩 시 해당 모드를 우선 사용합니다. NVENC는 ICQ를 지원하지 않기 때문에 무시됩니다.",
+    "transcoding_constant_rate_factor": "고정 비트율 계수 (-crf)",
+    "transcoding_constant_rate_factor_description": "영상 품질 수준입니다. 일반값: H.264=23, HEVC=28, VP9=31, AV1=35. 값이 낮을수록 품질은 좋지만 파일이 커집니다.",
+    "transcoding_disabled_description": "모든 비디오 트랜스코딩 비활성화 (일부 클라이언트에서 재생 오류 발생 가능)",
+    "transcoding_encoding_options": "인코딩 옵션",
+    "transcoding_encoding_options_description": "인코딩에 사용할 코덱, 해상도, 품질 등을 설정합니다.",
+    "transcoding_hardware_acceleration": "하드웨어 가속",
+    "transcoding_hardware_acceleration_description": "실험적 기능입니다. 속도는 빠르지만 동일 비트레이트 기준으로 품질이 낮을 수 있습니다.",
+    "transcoding_hardware_decoding": "하드웨어 디코딩",
+    "transcoding_hardware_decoding_setting_description": "인코딩뿐 아니라 디코딩도 하드웨어 가속을 사용합니다. 일부 영상에는 작동하지 않을 수 있습니다.",
+    "transcoding_hevc_codec": "HEVC 코덱",
+    "transcoding_max_b_frames": "최대 B-프레임 수",
+    "transcoding_max_b_frames_description": "값이 높을수록 압축 효율은 좋아지지만 인코딩 속도는 느려집니다. 구형 장치에서는 호환되지 않을 수 있습니다. 0은 비활성화, -1은 자동 설정",
+    "transcoding_max_bitrate": "최대 비트레이트",
+    "transcoding_max_bitrate_description": "최대 비트레이트를 설정하면 품질을 약간 희생하면서 예측 가능한 파일 크기를 유지할 수 있습니다. 720p 기준 예시: VP9/HEVC=2600kbit/s, H.264=4500kbit/s. 0은 비활성화.",
+    "transcoding_max_keyframe_interval": "최대 키프레임 간격",
+    "transcoding_max_keyframe_interval_description": "키프레임 간 최대 프레임 수를 설정합니다. 값이 낮을수록 압축 효율은 떨어지지만 탐색 속도와 빠른 움직임의 화질이 개선됩니다. 0은 자동 설정.",
+    "transcoding_optimal_description": "목표 해상도를 초과하거나 허용되지 않은 포맷의 비디오",
+    "transcoding_policy": "트랜스코딩 정책",
+    "transcoding_policy_description": "비디오를 트랜스코딩할 조건을 설정합니다.",
+    "transcoding_preferred_hardware_device": "우선 하드웨어 장치",
+    "transcoding_preferred_hardware_device_description": "VAAPI 및 QSV 전용. 트랜스코딩에 사용할 dri 노드를 지정합니다.",
+    "transcoding_preset_preset": "프리셋 (-preset)",
+    "transcoding_preset_preset_description": "압축 속도 설정. 느릴수록 품질과 압축률이 향상됩니다. VP9은 'faster' 이상 속도 무시",
+    "transcoding_reference_frames": "참조 프레임 수",
+    "transcoding_reference_frames_description": "압축 시 참조할 프레임 수입니다. 높을수록 압축 효율은 향상되지만 속도는 저하됩니다. 0은 자동 설정",
+    "transcoding_required_description": "허용 포맷이 아닌 영상만 트랜스코딩",
+    "transcoding_settings": "영상 트랜스코딩 설정",
+    "transcoding_settings_description": "트랜스코딩할 영상과 처리 방식 관리",
+    "transcoding_target_resolution": "목표 해상도",
+    "transcoding_target_resolution_description": "해상도가 높을수록 디테일은 보존되지만 인코딩 시간 증가, 파일 크기 증가, 앱 반응 저하 가능",
```

```suggestion
    "transcoding_target_resolution_description": "해상도가 높을수록 디테일은 보존되지만 인코딩 시간과 파일 크기가 증가하여 앱의 반응 속도가 느려질 수 있습니다.",
```
Since honorific language is used throughout, I’m maintaining it consistently for coherence.